extra/replace.c

```diff
@@ -173,6 +173,7 @@ register char **argv[];
 	break;
       case 'V':
 	version=1;
+    // fallthrough
```

Comment:
indentation

Reply:
Uh, the file is inconsistent here - at most places, it indents code using 2 spaces, but at this location, it uses tabs for some parts, and spaces for others... My original indendation matched the above line, but using spaces, not tabs, with a different tab size... I changed it to use the correct amount of spaces.